src/main/java/org/jenkinsci/test/acceptance/FallbackConfig.java

```diff
@@ -97,6 +100,14 @@ private WebDriver createWebDriver(TestName testName) throws IOException {
 
             profile.setPreference(LANGUAGE_SELECTOR, "en");
 
+            //
+            // When Tests are executed with a "large" number of plugins,
+            // the job configure page may display a script timeout in FireFox.
+            // We adjust this to be more lenient.
+            //
+            profile.setPreference(DOM_MAX_SCRIPT_RUN_TIME, 600);
```

Comment:
Please use relative time units as in https://github.com/jenkinsci/acceptance-test-harness/pull/253/files#diff-c21cecbd99df644c70b6d592e67a1a7dR196.